x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/lens/parser.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Plugin } from 'unified';
+import { RemarkTokenizer } from '@elastic/eui';
+
+export const LensParser: Plugin = function () {
+  const Parser = this.Parser;
+  const tokenizers = Parser.prototype.blockTokenizers;
+  const methods = Parser.prototype.blockMethods;
+
+  const tokenizeLens: RemarkTokenizer = function (eat, value, silent) {
+    if (value.startsWith('!{lens') === false) return false;
```

Comment:
I like this format. That's how I started doing the timeline's plugin. We changed it back to this format `[title](url)` because we wanted to be shown also on external services like ServiceNow. @XavierM What do you think of that?